realm/realm-library/proguard-rules.pro

```diff
@@ -1,5 +1,6 @@
 -keep class io.realm.annotations.RealmModule
 -keep @io.realm.annotations.RealmModule class *
+-keep class ** extends io.realm.RealmObject { *; }
```

Comment:
I am a bit sad that we have to reintroduce this as a lot of gymnastics we do with the Mediator classes was to allow obfuscation of model classes, but I'm guessing as long as we are using reflection for the fields there is no way around it :disappointed: